website/docs/docs/use-dbt-semantic-layer/set-dbt-semantic-layer.md

```diff
@@ -12,9 +12,11 @@ With the dbt Semantic Layer, you'll be able to centrally define business metrics
 Before you set up the dbt Semantic Layer, make sure you meet the following:
 
 <Snippet src="sl-prerequisites" />
+- After installing the dbt metrics package and updating the `packages.yml` file, make sure you run at least one model.
 
 <Snippet src="sl-considerations-banner" />
 
+Note: please have at least one model run after installing dbt metrics package and updating packages.yml 
```

Comment:
hey @Lana-hl great pr and so glad i got your first contribution 😍 ! i love this callout and thank you for linking the zendesk. so true and what do you think about adding it as a note under the 'prerequisites' section? that way the user is aware that it's a must. 

I'm suggesting removing this here so it's up under the prepreqs - wdyt

```suggestion
```

Reply:
@mirnawong1 looks good! Thank you so much!